{{ cookiecutter.app_name }}/{{ cookiecutter.app_name }}/config.py

```diff
@@ -0,0 +1,25 @@
+import os
+from typing import Literal, Optional
+
+from pydantic import BaseSettings, HttpUrl
+
+app_env = os.environ.get("APP_ENV", "dev")
+
+
+class Settings(BaseSettings):
+    app_name: str = "{{ cookiecutter.app_name }}"
+    app_env: Literal["production", "staging", "dev", "test"] = "dev"
+    release_version: Optional[str] = None
+    log_level: Literal["CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"] = "ERROR"
+
+    # Sentry
+    sentry_dsn: Optional[HttpUrl] = None
+
+    # Prometheus
+    prometheus_port: int = 8081
```

Comment:
and then have `if settings.prometheus_enabled` around starting HTTP server, to avoid starting it during tests or w/e
```suggestion
    prometheus_enabled: bool = True
    prometheus_port: int = 8081
```

Reply:
added